src/main/java/webApp/Response.java

```diff
@@ -7,15 +7,25 @@
 
     private final String message;
     private final String content;
+    private final Integer id;
```

Comment:
Why does the response class require the ID field, for what particular need?

Reply:
To report the id of a survey that is created.